src/main/java/socialite/commons/core/Messages.java

```diff
@@ -9,5 +9,7 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
-
+    public static final String MESSAGE_INVALID_HANDLE = "Invalid platform type given for p/\n "
+            + "SociaLite currently supports Facebook, Instagram, Telegram, Tiktok & Twitter.";
+    public static final String MESSAGE_EMPTY_TAG = "Tag must not be empty.";
```

Comment:
Also, I think line 12-14 should be put under `FindCommand` as a static field instead of in `Messages`! I think the messages here are the ones that can be used by multiple commands.

Reply:
Should we also do a `MESSAGE_EMPTY_HANDLE` likewise?